pkg/action/install_test.go

```diff
@@ -104,85 +102,15 @@ func TestName(t *testing.T) {
 	}
 	is.Equal("fleet", name)
 
-	// --generate-name while specifying a name
-	instAction = installAction(t)
-	instAction.GenerateName = true
-	chart = buildChart()
-	_, err = instAction.Name(chart, []string{"name1", "chart-uri"})
-	if err == nil {
-		t.Fatal(err)
-	}
-	is.Equal("cannot set --generate-name and also specify a name", err.Error())
-
 	// no name or annotations present
 	instAction = installAction(t)
 	instAction.ReleaseName = ""
 	chart = buildChart()
 	_, err = instAction.Name(chart, []string{"chart-uri"})
-	if err == nil {
-		t.Fatal("expected an error")
-	}
-	is.Equal("must either provide a name, set the correct chart annotations, or specify --generate-name", err.Error())
-}
-
-func TestNameGenerateName(t *testing.T) {
-	is := assert.New(t)
-	instAction := installAction(t)
-
-	instAction.ReleaseName = ""
-	instAction.GenerateName = true
-	chart := buildChart()
-
-	tests := []struct {
-		Name         string
-		Chart        string
-		ExpectedName string
-	}{
-		{
-			"local filepath",
-			"./chart",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-		{
-			"dot filepath",
-			".",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-		{
-			"empty filepath",
-			"",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-		{
-			"packaged chart",
-			"chart.tgz",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-		{
-			"packaged chart with .tar.gz extension",
-			"chart.tar.gz",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-		{
-			"packaged chart with local extension",
-			"./chart.tgz",
-			fmt.Sprintf("chart-%d", time.Now().Unix()),
-		},
-	}
-
-	for _, tc := range tests {
-		tc := tc
-		t.Run(tc.Name, func(t *testing.T) {
-			t.Parallel()
-
-			name, err := instAction.Name(chart, []string{tc.Chart})
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			is.Equal(tc.ExpectedName, name)
-		})
+	if err != nil {
+		t.Fatal(err)
 	}
+	is.Equal("fleet", name)
```

Comment:
```suggestion
	is.Equal("hello", name)
```

It should be hello, the chart was created with `buildchart()`, not with `buildchart(withFallbackAnnot())`. Together with the other change, makes everything peachy :).